### ISSUE TITLE
ElasticSearch Driver: The first argument must be one of type string, Buffer, ArrayBuffer, Array, or Array-like Object. Received type undefined

### ISSUE THREAD

**Issue by chad-codeworkshop** [cube-js/cube.js#614]
**Describe the bug**
When Running the elasticsearch driver in the latest version I get the following:
The first argument must be one of type string, Buffer, ArrayBuffer, Array, or Array-like Object. Received type undefined

**To Reproduce**
Use ElasticSearch Cloud driver with demo data and CubeJS playground

**Expected behavior**
No error on requesting data from elastic

**Screenshots**

**Version:**
0.19.13

**Additional context**

**Comment by chad-codeworkshop** [comment]
This issue is caused by passing empty strings in Client initializer for ElasticSearch

**Comment by chad-codeworkshop** [comment]
Should maybe have a graceful way of notifying the user?

**Comment by paveltiunov** [comment]
@chad-codeworkshop Hey Chad! Do you mean we need add sanity check to see if arguments passed? I believe it'd be helpful.

**Comment by chad-codeworkshop** [comment]
> @chad-codeworkshop Hey Chad! Do you mean we need add sanity check to see if arguments passed? I believe it'd be helpful.

Yeah, i spent about an hour tracking down the issue and it was something as simple as environment variables not being set, so the elastic Client failed

I will start having a look at how to implement this in the coming week
